themes/default/content/registry/packages/aws/how-to-guides/bucketv2-migration.md

```diff
@@ -0,0 +1,2423 @@
+---
+title: "BucketV2 Migration Guide"
+h1: "Migrating from aws.s3.Bucket to aws.s3.BucketV2"
+meta_desc: Practitioner level instructions for migrating from aws.s3.Bucket to aws.s3.BucketV2 resources
+layout: package
+---
+
+In the upcoming AWS Classic major release (v7), `aws.s3.Bucket` will be discontinued in favor of `BucketV2`. Users of
+`aws.s3.Bucket` resource are encouraged to migrate early. The migration is straightforward for simple use cases but
+requires some care for advanced configurations. Specifically `BucketV2` inputs such as `policy` and `accelerationStatus`
```

Comment:
nit: I'm not sure it's fair to call them advanced use cases. Things like enforcing secure transport or versioning are best practices for S3 (see https://docs.aws.amazon.com/AmazonS3/latest/userguide/security-best-practices.html).